config/kafkaconnect/config.go

```diff
@@ -0,0 +1,57 @@
+package kafkaconnect
+
+import (
+	"time"
+
+	"github.com/crossplane/upjet/pkg/config"
+	"github.com/upbound/provider-aws/config/common"
+)
+
+// Configure adds configurations for the kafkaconnect group.
+func Configure(p *config.Provider) {
+	p.AddResourceConfigurator("aws_mskconnect_connector", func(r *config.Resource) {
+		// This will always refer to a Cluster in the kafka api group, if it refers to any managed resource at all,
+		// but which property from the status of that cluster to use depends on the authentication mechanism chosen.
+		delete(r.References, "kafka_cluster.apache_kafka_cluster.bootstrap_servers")
+		r.References["kafka_cluster.apache_kafka_cluster.vpc.security_groups"] = config.Reference{
+			TerraformName:     "aws_security_group",
+			SelectorFieldName: "SecurityGroupSelector",
+			RefFieldName:      "SecurityGroupRefs",
+		}
+		r.References["kafka_cluster.apache_kafka_cluster.vpc.subnets"] = config.Reference{
+			TerraformName:     "aws_subnet",
+			SelectorFieldName: "SubnetSelector",
+			RefFieldName:      "SubnetRefs",
```

Comment:
The same situation applies here as with the `security_groups` reference above.

Reply:
https://github.com/crossplane-contrib/provider-upjet-aws/pull/1162#discussion_r1535511160